### DISCUSSION TITLE
Unknown command:"_BED_MESH_CALIBRATE" (whe creating bed mesh)

### DISCUSSION THREAD

**Discussion by cyaneo** [bassamanator/Sovol-SV06-firmware#160] [General]
Hello,

I'm on "One-Stop-Shop Klipper Configuration", "Configure Your Printer"
Point 1 to 5 working fine, but  on point 6
...
6. Create a bed mesh.
`DO_CREATE_MESH`
 `SAVE_CONFIG` (once completed)

I get the error
`Unknown command: "_BED_MESH_CALIBRATE"`

What did I missed out?

**Comment by bassamanator** [comment]
You made a mistake in the downloading process.

**Comment by cyaneo** [comment]
I did Method 1: Clone the Repository

💡 Make sure git is installed (sudo apt update && sudo apt install git).

    cd ~/printer_data/config
    Empty entire ~/printer_data/config folder.
        In linux, you can delete files via rm fileName and directories via rmdir directoryName.
        In linux, you can list files and folders via ls -lah.
    git clone -b sv06-plus --single-branch https://github.com/bassamanator/Sovol-SV06-firmware.git . ⚠️ Don't miss the period!

with no errors.

What did I wrong?
![path01](https://github.com/bassamanator/Sovol-SV06-firmware/assets/8118394/2332d0dd-87c5-445d-9367-9c3b5302a654)
![path02](https://github.com/bassamanator/Sovol-SV06-firmware/assets/8118394/b0a5fb11-735b-4ec0-a62c-e0c179ab4b1e)

**Comment by bassamanator** [comment]
You have more files in your `config` folder than are included in my configuration, likely one of them is overwriting the variable/macro in question. If I had to guess, the problem lies in `print_area_bed_mesh.cfg`.

**Reply by cyaneo** [reply]
you make my day! - I left old "Klipper Printer Additions (Vick) pack" files and some lines in printer.cfg behind - remove, restart, ready to go!
Thank you!

**Reply by bassamanator** [reply]
Great. Please make my post as the answer, and close the discussion.

**Comment by cyaneo** [comment]
you make my day! - I left old "Klipper Printer Additions (Vick) pack" files and some lines in printer.cfg behind - remove, restart, ready to go!
Thank you!
